cmd/harmony/main.go

```diff
@@ -811,9 +840,7 @@ func setupSyncService(node *node.Node, host p2p.Host, hc harmonyconfig.HarmonyCo
 	node.RegisterService(service.Synchronize, s)
 
 	d := s.Downloaders.GetShardDownloader(node.Blockchain().ShardID())
-	if hc.Sync.Downloader {
```

Comment:
why remove this condition? This is the condition to enable stream sync (@JackyWYX can confirm.)

Reply:
Was removed accidentally. Restored it.